web/forms.py

```
@@ -829,17 +835,15 @@
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        # Order subjects by name
-        self.fields["category"].queryset = Subject.objects.all().order_by("order", "name")
+# Order subjects by 'order' first, then alphabetically by 'name'
 
     def clean_video_url(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_video_url`

(ANN201)